lib/components/narrative/default/default-itinerary.js

```diff
@@ -23,6 +28,11 @@ const { isBicycle, isMicromobility, isTransit } = coreUtils.itinerary
 
 // Styled components
 const LegIconWrapper = styled.div`
+  display: inline-block;
```

Comment:
Has this CSS be duplicated?

Reply:
This was added by merging dev into this branch, so I'm not quite sure why it's appearing as a change!

Reply:
Addressed in 88c0af89a48df3baed5064b7a4dd86720c3ca06d